### ISSUE TITLE
Absolute path support for imports

### ISSUE THREAD

**Issue by kkomaz** [blockstack/blockstack-browser#825]
From a development perspective, support for absolute path would be great.  With absolute path, we would no longer have to do deep nesting when importing utility functions or separate components.  Example below

```
import { DROPBOX } from '../../../account/utils/index'
```
to
```
import { DROPBOX } from 'js/account/utils/index'
```

**Comment by kkomaz** [comment]
@guylepage3 This is an enhancement I can work on if you like the idea.

**Comment by larrysalibra** [comment]
@kkomaz is this something that's possible?

**Comment by kkomaz** [comment]
Yep!  Just need some minor configurations via
https://github.com/tleunen/babel-plugin-module-resolver

**Comment by yknl** [comment]
This would be nice to have!

**Comment by guylepage3** [comment]
Nice @kkomaz.. 😄

**Comment by larrysalibra** [comment]
Would be nice to have. 

Keep in mind eslint and flow need to continue working along with the respective plugins in our editors. It looks like getting all of these to work requires adding a bunch of additional dependencies both to the project and to the plugins our editors use - not sure how well they are maintained.

I'm in general hesitant to add too many dependencies that aren't widely used, but feel free to give it a go.

If it works well happy to merge it.

**Comment by dashaxiong** [comment]
It looks like this issue should be closed, as per issue #828 and @aulneau commit 403ea67b697d

**Comment by wbobeirne** [comment]
^ Indeed!
